packages/client/src/types/store.ts

```diff
@@ -164,6 +164,10 @@ export interface FirestoreData {
   };
   [OrgSubCollection.SlotsByDay]: { [monthStr: string]: SlotsByDay } | null;
   [OrgSubCollection.Attendance]: { [slotId: string]: SlotAttendnace };
+  [Collection.PublicOrgInfo]: {
+    [organization: string]: Pick<OrganizationData, "displayName"> &
+      Pick<OrganizationData, "location">;
+  };
```

Comment:
This goes hand in hand with my suggestion: The `publicOrgInfo` is a subset of `OrganizationData`. Also, what I've lately discovered, the long `Pick` subset can be written (shortly) like this: `Pick<OrganizationData, "displayName" | "location">`